geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build

```diff
@@ -138,15 +138,17 @@ def main() -> None:
             sys.exit(e.returncode)
 
     if args.fast_reload:
-        for service in (
-            subprocess.run(
+        services = [
+            service
+            for service in subprocess.run(
                 ["docker-compose", "ps", "--services", "--all"], stdout=subprocess.PIPE, check=True
             )
             .stdout.decode()
             .splitlines()
-        ):
-            if not service.startswith("redis"):
-                run(args, ["docker-compose", "rm", "--stop", "--force", service], check=True)
+            if not service.startswith("redis")
+        ]
+
+        run(args, ["docker-compose", "rm", "--stop", "--force"] + services, check=True)
```

Comment:
You do not remove volumes, not sure this will really update the config.

For example I'm used to run:
```
docker-compose rm --stop -f -v config && docker-compose up -d --force-recreate geoportal
```

Reply:
Yes, it updates the config :-)